src/core/worker.js

```diff
@@ -726,6 +726,7 @@ class WorkerMessageHandler {
           acroFormRef,
           acroForm,
           xfaData,
+          useXrefStream: xref.topDict?.get("Type")?.name === "XRef",
```

Comment:
It's not immediately clear, when looking at this, what kind of data you're checking for here.
Please note that we have a helper function for this very case, [see here](https://github.com/mozilla/pdf.js/blob/7cfcd00a385314a05e0ff247d80262200a0e1556/src/core/primitives.js#L391-L395), which allows us to write this as follows.
```suggestion
          useXrefStream: isDict(xref.topDict, "XRef"),
```